cudax/include/cuda/experimental/__memory_resource/any_resource.h

```diff
@@ -53,6 +53,16 @@ namespace cuda::experimental::mr
 template <class _Ty, class _Uy = _CUDA_VSTD::remove_cvref_t<_Ty>>
 _LIBCUDACXX_INLINE_VAR constexpr bool __is_basic_any_resource = false;
 
+//! @rst
+//! .. _cudax-memory-resource-basic-any-resource:
+//!
+//! Base class for a type erased owning wrapper around a memory resource
+//! ---------------------------------------------------------------------
+//!
+//! ``basic_any_resource`` abstracts the differences between a resource and an async resource away, allowing efficient
+//! interoperability between the two.
```

Comment:
i'm not sure what this is referring to. `any_resource` and `any_async_resource` are different types with different interfaces. the differences haven  't been abstracted away.